Add configurable mouse sensitivity

dexgs opened the pull request:
The `--mouse-sensitivity`/`-s` command line option allows the user to set the value `g_mouseSensitivity` which is used to scale the mouse movement in `wlserver_mousemotion`.

This works well for reducing sensitivity: sub-pixel increments taken into account to avoid jerky cursor movement as a result of rounding.

However, increasing sensitivity results in a loss of precision proportional to the multiplier since it seems mouse movements are only ever given in terms of pixels (i.e. a multiplier of `2.0` causes 2 pixels to be the shortest distance the cursor can be moved. If there is a way to get around this, please let me know).

This change is a possible workaround for #196 and #244, but the main motivation is to allow forcing mouse sensitivity scaling on games that don't respect the host mouse settings or otherwise misbehave with respect to cursor speed.

dexgs commented on src/wlserver.cpp:
Thanks

Joshua-Ashton commented:
Why not just use `wlserver_perform_rel_pointer_motion` instead of implementing the same thing in wlserver_mousemotion. We can probably just remove wlserver_mousemotion at this point.

dexgs commented:
Okay, I've changed the implementation. I'm not sure how you want to move things around in order to call into wlserver_perform_rel_pointer_motion` from outside wlserver, so I just replaced the body of wlserver_mousemotion with a call to it for now.
